src/app/about/page.client.tsx

```diff
@@ -0,0 +1,151 @@
+    'All Members',
+    'Executive',
+    'Product/UIUX',
+    'Developers',
+    'Business Development',
+    'Data',
+  ];
+
+  const filterMembers = (role: string) => {
+    if (role === 'All Members') return teamMemberData;
+    return teamMemberData.filter((member) => member.role.includes(role));
+  };
+
+  const filteredMembers = filterMembers(filter);
+
+  const displayedMembers = filteredMembers.slice((page - 1) * 6, page * 6);
+  const totalPages = Math.ceil(filteredMembers.length / 6);
+
+  const formatPageNumber = (number: number | null) => {
+    if (number === null) return '00';
+    return number.toLocaleString('en-US', { minimumIntegerDigits: 2 });
+  };
+
+  const renderRoleButtons = roles.map((role) => (
+    <Button
+      key={role}
+      className={`about-page__filter-button ${filter === role ? 'about-page__filter-button--active' : ''}`}
+      onClick={() => {
+        setFilter(role);
+        setPage(1);
+      }}
+    >
+      {role}
+    </Button>
+  ));
+
+  return (
+    <Container className="about-page">
+      <Script id="jsonLD_team-member" type="application/ld+json">
+        {JSON.stringify(teamMemberJson)}
+      </Script>
+      <Box className="about-page__heading-container">
+        <Typography className="page-heading about-page__heading" variant="h1">
+          Meet Our Team
+        </Typography>
+        <Typography className="about-page__text" variant="body1">
+          We are volunteer developers, designers, project managers, engineers,
+          marketers, organizers, strategists, and citizens committed to our
+          vision, and driven by our mission.
+        </Typography>
+      </Box>
+      <Box className="about-page__content-container">
+        <Box className="about-page__filter">{renderRoleButtons}</Box>
+        <Box className="about-page__content-wrapper">
+          <Box className="about-page__card-container">
+            {displayedMembers.map((teamMember: TeamMember) => (
+              <TeamBioCard key={teamMember.name} data={teamMember} />
+            ))}
+          </Box>
+          <Pagination
+            count={totalPages}
+            page={page}
+            onChange={(event, value) => setPage(value)}
+            className="about-page__pagination"
+            variant="text"
+            size="large"
+            color="primary"
+            shape="rounded"
+            renderItem={(item) => (
+              <PaginationItem
+                {...item}
+                page={
+                  item.type === 'page' && item.page !== null
+                    ? formatPageNumber(item.page)
+                    : item.page
+                }
+              />
+            )}
+          />
+        </Box>
+      </Box>
+      <Box className="join-us-cta-card">
+        <hr className="join-us-cta-card__divider" />
+        <Box className="join-us-cta-card__text-container">
+          <Typography className="join-us-cta-card__title" variant="body1">
+            Want to be a part?
```

Comment:
More of a note for the design team; grammatically this seems like it should be something like:

```
Want to be a part of our team?
```

Reply:
Pinged the Design team here on Discord about this:

https://discord.com/channels/1211548956719317022/1219462328710598757/1259655412035223603

Reply:
addressed in [2d1f12a](https://github.com/CodeForBc/codeforbc-web/pull/51/commits/2d1f12a46e912fd961e9b3139ffe029c35482ac8)